### ISSUE TITLE
Rule `no-extra-spacing-attrs` removing other attributes.

### ISSUE THREAD

**Issue by maheshbansod** [yeonjuan/html-eslint#137]
Noticed that if an attribute value with special characters (e.g. `_` / `{` ) or some invalid value appears in HTML and the rule `@html-eslint/no-extra-spacing-attrs` is active then eslint (with `--fix`) strips off everything after the attribute name.

Example:
HTML
```
<a target=”_blank” a b c d e f>
link
</a>

```
eslint config:
```
{
	"plugins": ["@html-eslint"],
	"overrides": [{
		"files": ["*.html"],
		"parser": "@html-eslint/parser",
		"rules": {
			"@html-eslint/no-extra-spacing-attrs": ["warn"]
		}
	}]
}
```
Observed result HTML:
```
<a target>
link
</a>
```
Edit: I've not quite figured out the actual problem.. after changing it to `target="blank"` (i.e. without special characters), I still get the same result - the other attributes are removed - this doesn't happen in case of `href` attribute.

**Comment by maheshbansod** [comment]
You're right, @yeonjuan , the `"` was a typo. Embarassd that I didn't notice it sooner :smile: 
I guess it's breaking for special characters then.

**Comment by yeonjuan** [comment]
It's bugs in [es-html-parser](https://github.com/yeonjuan/es-html-parser).
The parser produces incorrect location information.

<img width="904" alt="스크린샷 2023-07-21 오전 12 32 25" src="https://github.com/yeonjuan/html-eslint/assets/41323220/8b0bab1a-b62b-42a8-a8ec-55542789d70b">

**Comment by yeonjuan** [comment]
@maheshbansod It's fixed and released! (v1.9.1) Thanks!
